### ISSUE TITLE
unmarshal errors when using latest grafana/tempo image

### ISSUE THREAD

**Issue by kaynenotkanye** [mnadeem/boot-opentelemetry-tempo#21]
The latest grafana/tempo image keeps failing to start due to unmarshal errors related to the tempo.yaml.

```
$ docker logs boot-opentelemetry-tempo_tempo_1
failed parsing config: failed to parse configFile /etc/tempo.yaml: yaml: unmarshal errors:
  line 38: field bloom_filter_false_positive not found in type wal.Config
  line 39: field index_downsample not found in type wal.Config
```

As a workaround, I was able to downgrade the image in docker-compose to grafana/tempo:0.5.0.  Issue seems to happen with grafana/tempo:0.6.0 and higher.

**Comment by mnadeem** [comment]
https://github.com/grafana/tempo/releases This would be helpful, I will update the images with correct release

**Comment by mnadeem** [comment]
Take care, bloom_filter_false_positive and index_downsample fixed the tempo version to 0.7.0

Thanks for opening this
